cpp/include/cuml/fil/fil.h

```diff
@@ -187,6 +187,10 @@ void init_sparse(const cumlHandle& h, forest_t* pf, const int* trees,
 void from_treelite(const cumlHandle& handle, forest_t* pforest,
                    ModelHandle model, const treelite_params_t* tl_params);
 
+
+void from_multi_treelites(const cumlHandle& handle, forest_t* pforest,
```

Comment:
I think it's better to have a separate function that merges N treelite models into one. Then we can import the resulting model into FIL using `from_treelite()`.